contracts/modules/STO/POLYCappedSTOFactory.sol

```diff
@@ -0,0 +1,90 @@
+pragma solidity ^0.5.0;
+
+import "../ModuleFactory.sol";
+import "../../proxy/POLYCappedSTOProxy.sol";
+import "../../libraries/Util.sol";
+import "../../interfaces/IBoot.sol";
+
+/**
+ * @title Factory for deploying POLYCappedSTO module
+ */
+contract POLYCappedSTOFactory is ModuleFactory {
+
+    address public logicContract;
+    /**
+     * @notice Constructor
+     * @param _setupCost Setup cost of the module
+     * @param _usageCost Usage cost of the module
+     * @param _logicContract Contract address that contains the logic related to `description`
+     * @param _polymathRegistry Address of the Polymath registry
+     */
+    constructor(
+        uint256 _setupCost,
+        uint256 _usageCost,
+        address _logicContract,
+        address _polymathRegistry
+    )
+        public
+        ModuleFactory(_setupCost, _usageCost, _polymathRegistry)
+    {
+        require(_logicContract != address(0), "address 0x not allowed");
+        logicContract = _logicContract;
+        version = "1.0.0";
```

Comment:
This module requires version 2.0+ of ST. Let's bump it to 3.0.0 directly as that is where it will be merged.

Reply:
Ok - I wasn't sure if the factory versions should be linked to the Core/ST release or if its version should be independent so I started at 1.0.0. I'll change to 3.0.0